### PR TITLE
feature/1756 - Add Contour Airlines (VTE)

### LINKED ISSUE #1756: Add Contour Aviation [VTE]
Contour Aviation- needed for KBWI.

### PR DESCRIPTION
**Pull request by Hawaiicoder**
Resolves #1756 

The purpose of this pull request is to add Contour Airlines for the KBWI airport project.

### CODE REVIEW

File: assets/airlines/vte.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+            "###"
+        ]
+    },
+    "fleets": {
+        "default": [
+            ["E135", 10],
+            ["E145", 2]
+        ]
+    }
+}
```
Comment (erikquinn) [inline assets/airlines/vte.json:17]: always need an empty line at the end of the file.

**Review by erikquinn — DISMISSED** [review]
Please use @openscope/airport-reviewers for all PRs on any changes that only involve airport/airline changes.

**Review by Fechulo — CHANGES_REQUESTED** [review]
Looks like they might have more than just Embraers:

> The company operates a diverse fleet of aircraft for private charter, ranging in size from 8 passenger light jets to three cabin ultra long-range business jets.

Taken from their [website](https://contourairlines.com/corporate/about-us)

According to [FlightRadar24](https://www.flightradar24.com/data/airlines/lf-vte/fleet) they have a few others as well

**Comment by Hawaiicoder** [comment]
@erikquinn Understood on both- thanks for letting me know. 
@Fechulo Data was taken from Planespotters but I will be updating according to the FR24 data.

**Review by Fechulo — APPROVED** [review]

**Comment by Hawaiicoder** [comment]
Is there anything else I need to do for this to be merged?

**Comment by erikquinn** [comment]
no, it just takes time. they have to go in one at a time, and this one was not at the top of the priority list. once you get an approved review, it will only be a matter of time, and it should not be adjusted any further unless coordinated otherwise.

**Comment by Hawaiicoder** [comment]
Good to know! Thanks!

**Pull request merged**